test/unit-tests/function/arithmetic/subtractScalar.test.js

```diff
@@ -0,0 +1,267 @@
+// test subtract
+import assert from 'assert'
+
+import approx from '../../../../tools/approx.js'
+import math from '../../../../src/defaultInstance.js'
+import Decimal from 'decimal.js'
+const { subtract, BigNumber } = math
+
+describe('subtractScalar', function () {
+  it('should subtract two numbers correctly', function () {
+    assert.deepStrictEqual(subtract(4, 2), 2)
```

Comment:
I may be overlooking a test, but I think we should also test `subtractScalar` with more than 2 values.

Reply:
Ok will look into it definitely.

Reply:
done